api/v1alpha1/flowcollector_types.go

```diff
@@ -38,6 +38,7 @@ const (
 // https://github.com/kubernetes-sigs/controller-tools/issues/622
 
 // FlowCollectorSpec defines the desired state of FlowCollector
+// +k8s:conversion-gen=false
```

Comment:
@msherif1234 can you please take a look at this ?

We had to put `// +k8s:conversion-gen=false` on top of `FlowCollectorSpec` because we need to manage the `Exporters` array conversion.

Without that, the following generated functions fails:
- `autoConvert_v1alpha1_FlowCollectorSpec_To_v1beta1_FlowCollectorSpec`
- `autoConvert_v1beta1_FlowCollectorSpec_To_v1alpha1_FlowCollectorSpec`

Is there a better way to manage this array conversion without overriding the entire `FlowCollectorSpec` one ?